spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/authenticated_user'
+
+module Api
+  module V1
+    describe BooksController, type: :controller do
+      include_context 'Authenticated User'
+
+      describe 'GET #index' do
+        context 'When fetching all the books' do
+          let(:books) { create_list(:book, 5) }
+          before do
+            get :index
+          end
+          it 'responses with the books json' do
+            expected = ActiveModel::Serializer::CollectionSerializer.new(
+              books, each_serializer: BookSerializer
+            ).to_json
+            expect(response.body.to_json) =~ JSON.parse(expected)
+          end
+
+          it 'responses with status 200' do
+            expect(response).to have_http_status(:ok)
+          end
+        end
+      end
+
+      describe 'GET #show' do
+        context 'When fetching a Book' do
```

Comment:
Be careful, we only have here the "good" tests.
Do not forget to make the "bad" ones, like testing when asking for a not existing book. They are as important as the anothers.

Reply:
Understood, I will add the "bad" scenarios to this test